frontend/scripts/react-components/home/slider-section.component.jsx

```diff
@@ -0,0 +1,122 @@
+/* eslint-disable react/no-danger */
+import React from 'react';
+import PropTypes from 'prop-types';
+import Siema from 'react-siema';
+import cx from 'classnames';
+
+class SliderSection extends React.PureComponent {
+  static getPerPage() {
+    switch (true) {
+      case (window.innerWidth < 640): return 1;
+      case (window.innerWidth < 950): return 2;
+      default: return 3;
+    }
+  }
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      currentSlide: 0,
+      perPage: SliderSection.getPerPage()
```

Comment:
IMO this doesn't really make sense on state. Just use `const perPage = SliderSection.getPerPage();` on line 44 and it should be enough, and have the added bonus of working nicely on browser window resize